setup.py

```diff
@@ -20,15 +20,15 @@
 tests_require = [
     'check-manifest>=0.25',
     'coverage>=4.0',
-    'httpretty>=0.8.14',
+    'responses>=0.10.6',
```

Comment:
@lnielsen Do you see any strong issue that ``httpretty`` to ``responses`` might imply? License is Apache, seems to be well maintained, use is almost 1-1 match...

Reply:
No problem with Apache license.....only GPL and AGPL are an issue.